src/components/FileExtensionForm/FileExtensionForm.js

```diff
@@ -66,11 +66,7 @@ class FileExtensionFormComponent extends Component {
   };
 
   formatMultiSelect = ({ option, searchTerm }) => {
```

Comment:
```suggestion
  formatMultiSelect = ({ option, searchTerm }) => <OptionSegment searchTerm={searchTerm}>{option}</OptionSegment>;
```
Not sure about that. At your discretion ;)